configuration/README.md

```diff
@@ -13,14 +13,26 @@ For design details concerning value and instruction classification, see [/design
 
 Configuration is provided to `go-flow-levee` via YAML.
 
-Objects of interest are identified primarily via regexp. An empty regexp will match any string.
+Sources are identified by package, type, and field names.
+You may specify these with a combination of string literals or regexp.
```

Comment:
What if both "versions" of a matcher are given, e.g. a `Source` has both a `Package` and a `PackageRE`?

(I saw your code that produces an error if this occurs, but I think it could be good to mention it here as well.)

((Currently I think users may not even see the error string if they are running with `go vet`. We should consider printing the error when loading the config in `fieldpropagators` and `source`, or finding a way to have `go vet` itself output the error. This is out of scope for the current PR.))